app/api/views/api_view.py

```diff
@@ -47,7 +49,9 @@ def stun_tag(request):
         if type(time) != int:
             return badRequest()
 
-        date = datetime.fromtimestamp(time)
+        eastern = timezone('US/Eastern')
+
+        date = datetime.fromtimestamp(time).replace(tzinfo=pytz.utc).astimezone(tz=eastern)
```

Comment:
Ah I see why this is necessary. Cool, I didn't imagine this lol. Another option is we just convert to EST when sending but this is fine.

Reply:
Can't really convert timestamp to EST lol